tests/comments/if.js

```diff
@@ -59,3 +59,21 @@ else if (15) // comment
 /* comment */
 /* comment */ // comment
 true
+
+if (cond) 
+  console.log("db");
```

Comment:
Prefer creating new test files to appending to existing files. When the files get long (> 20-30 lines), the snapshots become difficult to read.